scrapy_fake_useragent/middleware.py

```diff
@@ -1,6 +1,10 @@
 import logging
 from fake_useragent import UserAgent
 
+from scrapy import signals
+from scrapy.downloadermiddlewares.retry import *
```

Comment:
Could you please replace a `*` import with importing only specific things needed in this module?https://stackoverflow.com/questions/2386714/why-is-import-bad Thanks!

Reply:
Oh sure! Of course, following PEP8 is important. wait a sec.